Sources/TransferMethod/SelectTransferMethodTypePresenter.swift

```diff
@@ -264,39 +262,36 @@ final class SelectTransferMethodTypePresenter {
         }
 
         for transferMethodType in result {
-            let transferMethodTypeDetail = transferMethodConfigurationKeyResult!
-                .populateTransferMethodTypeDetail(country: country,
-                                                  currency: currency,
-                                                  profileType: profileType,
-                                                  transferMethodType: transferMethodType)
+            let transferMethodTypeDetail = TransferMethodTypeDetail(currency, transferMethodType)
             transferMethodTypes.append(transferMethodTypeDetail)
         }
         view.transferMethodTypeTableViewReloadData()
     }
 
     private func loadCurrency(for country: String) {
         guard let firstCurrency = transferMethodConfigurationKeyResult?
-            .currencies(from: country).min(by: { $0.localized() < $1.localized() }) else {
-            view.showAlert(message: String(format: "no_currency_available_error_message".localized(),
-                                           country.localized()))
-            selectedCurrency = ""
-            view.countryCurrencyTableViewReloadData()
-            return
+            .currencies(from: country)?.min(by: { $0.code.localized() < $1.code.localized() })
```

Comment:
Use name instead of code.localized() as name now contains localized value for the code. Delete all the country and currency code=name pairs from localizable.strings.

Reply:
fixed